### DISCUSSION TITLE
PakcerSync error using packer

### DISCUSSION THREAD

**Discussion by ZichengWangdrzz** [L3MON4D3/LuaSnip#904] [General]
```
use({
	"L3MON4D3/LuaSnip",
	-- follow latest release.
	tag = "v<CurrentMajor>.*", -- Replace <CurrentMajor> by the latest released major (first number of latest release)
	-- install jsregexp (optional!:).
	run = "make install_jsregexp"
})
```

Just want to make sure I'm doing the right way through pakcer. The latest release I believe it is 1.2.1. What does **the first number of latest release** means>? It is 1 ,2 or 1.2.1? I tried several ways, they all went wrong. Please hellp me, thank you.

**Comment by L3MON4D3** [comment]
`1` should work..
Try commenting out the jsregexp-line

**Reply by no-vici** [reply]
@ZichengWangdrzz, 

You should commend the line starting with `tag = "v<CurrentMajor>.*"` as well for the latest update, then `:PackerSync`.

**Reply by ZichengWangdrzz** [reply]
@L3MON4D3 Do you mean commenting out the line "run = "make install_jsregexp""?

**Reply by ZichengWangdrzz** [reply]
@no-vici Sorry, I don't quite get it, what does it mean to commend the line? Can you please explain?

**Reply by ZichengWangdrzz** [reply]
@L3MON4D3 Since the line is optional , I just deleted them all and tried several times:
`
use({
"L3MON4D3/LuaSnip", -- follow latest release. tag = "v1.*", -- Replace by the latest released major (first number of latest release) })
`
Still got the same errors.

**Reply by L3MON4D3** [reply]
What errors are you getting?
And you can embed multiline-code with
````text
```lua
<code here>
```
````

**Reply by ZichengWangdrzz** [reply]
@L3MON4D3 Thank you for the note. It's an error generated by PackerSync:
 "✗ Failed to update L3MON4D3/LuaSnip"

```lua
use({
    "L3MON4D3/LuaSnip",
    -- follow latest release.
    tag = "v1.*" -- Replace <CurrentMajor> by the latest released major (first number of latest release)   
  })
```

**Reply by L3MON4D3** [reply]
So far the `use` looks correct :+1:
Can you get any more details? Maybe in packers logs?

**Reply by ZichengWangdrzz** [reply]
Sure thing. Here's the logs. 
```lua
-- Automatically generated packer.nvim plugin loader code

if vim.api.nvim_call_function('has', {'nvim-0.5'}) ~= 1 then
  vim.api.nvim_command('echohl WarningMsg | echom "Invalid Neovim version for packer.nvim! | echohl None"')
  return
end

vim.api.nvim_command('packadd packer.nvim')

local no_errors, error_msg = pcall(function()

_G._packer = _G._packer or {}
_G._packer.inside_compile = true

local time
local profile_info
local should_profile = false
if should_profile then
  local hrtime = vim.loop.hrtime
  profile_info = {}
  time = function(chunk, start)
    if start then
      profile_info[chunk] = hrtime()
    else
      profile_info[chunk] = (hrtime() - profile_info[chunk]) / 1e6
    end
  end
else
  time = function(chunk, start) end
end

local function save_profiles(threshold)
  local sorted_times = {}
  for chunk_name, time_taken in pairs(profile_info) do
    sorted_times[#sorted_times + 1] = {chunk_name, time_taken}
  end
  table.sort(sorted_times, function(a, b) return a[2] > b[2] end)
  local results = {}
  for i, elem in ipairs(sorted_times) do
    if not threshold or threshold and elem[2] > threshold then
      results[i] = elem[1] .. ' took ' .. elem[2] .. 'ms'
    end
  end
  if threshold then
    table.insert(results, '(Only showing plugins that took longer than ' .. threshold .. ' ms ' .. 'to load)')
  end

  _G._packer.profile_output = results
end

time([[Luarocks path setup]], true)
local package_path_str = "/Users/hugowang/.cache/nvim/packer_hererocks/2.1.0-beta3/share/lua/5.1/?.lua;/Users/hugowang/.cache/nvim/packer_hererocks/2.1.0-beta3/share/lua/5.1/?/init.lua;/Users/hugowang/.cache/nvim/packer_hererocks/2.1.0-beta3/lib/luarocks/rocks-5.1/?.lua;/Users/hugowang/.cache/nvim/packer_hererocks/2.1.0-beta3/lib/luarocks/rocks-5.1/?/init.lua"
local install_cpath_pattern = "/Users/hugowang/.cache/nvim/packer_hererocks/2.1.0-beta3/lib/lua/5.1/?.so"
if not string.find(package.path, package_path_str, 1, true) then
  package.path = package.path .. ';' .. package_path_str
end

if not string.find(package.cpath, install_cpath_pattern, 1, true) then
  package.cpath = package.cpath .. ';' .. install_cpath_pattern
end

time([[Luarocks path setup]], false)
time([[try_loadstring definition]], true)
local function try_loadstring(s, component, name)
  local success, result = pcall(loadstring(s), name, _G.packer_plugins[name])
  if not success then
    vim.schedule(function()
      vim.api.nvim_notify('packer.nvim: Error running ' .. component .. ' for ' .. name .. ': ' .. result, vim.log.levels.ERROR, {})
    end)
  end
  return result
end

time([[try_loadstring definition]], false)
time([[Defining packer_plugins]], true)
_G.packer_plugins = {
  LuaSnip = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/LuaSnip",
    url = "https://github.com/L3MON4D3/LuaSnip"
  },
  ["cmp-buffer"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/cmp-buffer",
    url = "https://github.com/hrsh7th/cmp-buffer"
  },
  ["cmp-path"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/cmp-path",
    url = "https://github.com/hrsh7th/cmp-path"
  },
  cmp_luasnip = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/cmp_luasnip",
    url = "https://github.com/saadparwaiz1/cmp_luasnip"
  },
  ["friendly-snippets"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/friendly-snippets",
    url = "https://github.com/rafamadriz/friendly-snippets"
  },
  harpoon = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/harpoon",
    url = "https://github.com/theprimeagen/harpoon"
  },
  ["lualine.nvim"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/lualine.nvim",
    url = "https://github.com/nvim-lualine/lualine.nvim"
  },
  ["nvim-cmp"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/nvim-cmp",
    url = "https://github.com/hrsh7th/nvim-cmp"
  },
  ["nvim-tree.lua"] = {
    config = { "\27LJ\2\n;\0\0\3\0\3\0\a6\0\0\0'\2\1\0B\0\2\0029\0\2\0004\2\0\0B\0\2\1K\0\1\0\nsetup\14nvim-tree\frequire\0" },
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/nvim-tree.lua",
    url = "https://github.com/nvim-tree/nvim-tree.lua"
  },
  ["nvim-treesitter"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/nvim-treesitter",
    url = "https://github.com/nvim-treesitter/nvim-treesitter"
  },
  ["nvim-web-devicons"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/nvim-web-devicons",
    url = "https://github.com/nvim-tree/nvim-web-devicons"
  },
  ["packer.nvim"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/packer.nvim",
    url = "https://github.com/wbthomason/packer.nvim"
  },
  playground = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/playground",
    url = "https://github.com/nvim-treesitter/playground"
  },
  ["plenary.nvim"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/plenary.nvim",
    url = "https://github.com/nvim-lua/plenary.nvim"
  },
  ["rose-pine"] = {
    config = { "\27LJ\2\n9\0\0\3\0\3\0\0056\0\0\0009\0\1\0'\2\2\0B\0\2\1K\0\1\0\26colorscheme rose-pine\bcmd\bvim\0" },
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/rose-pine",
    url = "https://github.com/rose-pine/neovim"
  },
  ["telescope.nvim"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/telescope.nvim",
    url = "https://github.com/nvim-telescope/telescope.nvim"
  },
  undotree = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/undotree",
    url = "https://github.com/mbbill/undotree"
  },
  ["vim-fugitive"] = {
    loaded = true,
    path = "/Users/hugowang/.local/share/nvim/site/pack/packer/start/vim-fugitive",
    url = "https://github.com/tpope/vim-fugitive"
  }
}

time([[Defining packer_plugins]], false)
-- Config for: nvim-tree.lua
time([[Config for nvim-tree.lua]], true)
try_loadstring("\27LJ\2\n;\0\0\3\0\3\0\a6\0\0\0'\2\1\0B\0\2\0029\0\2\0004\2\0\0B\0\2\1K\0\1\0\nsetup\14nvim-tree\frequire\0", "config", "nvim-tree.lua")
time([[Config for nvim-tree.lua]], false)
-- Config for: rose-pine
time([[Config for rose-pine]], true)
try_loadstring("\27LJ\2\n9\0\0\3\0\3\0\0056\0\0\0009\0\1\0'\2\2\0B\0\2\1K\0\1\0\26colorscheme rose-pine\bcmd\bvim\0", "config", "rose-pine")
time([[Config for rose-pine]], false)

_G._packer.inside_compile = false
if _G._packer.needs_bufread == true then
  vim.cmd("doautocmd BufRead")
end
_G._packer.needs_bufread = false

if should_profile then save_profiles() end

end)

if not no_errors then
  error_msg = error_msg:gsub('"', '\\"')
  vim.api.nvim_command('echohl ErrorMsg | echom "Error in packer_compiled: '..error_msg..'" | echom "Please check your config for correctness" | echohl None')
end
```

**Reply by L3MON4D3** [reply]
Are you sure those are the logs? Looks more like the file packer generates for actually loading the plugins..

**Reply by ZichengWangdrzz** [reply]
Ok, I don't understand what is the packer logs you refer to since I am new to nvim. Just got the first touch to this plugin, too. Can you tell me where to find the packers log, maybe?`

**Reply by L3MON4D3** [reply]
Always a good idea to search the plugins [github](https://github.com/wbthomason/packer.nvim#debugging) :P
You can find this path via `:lua =vim.fn.stdpath("cache")`

**Reply by ZichengWangdrzz** [reply]
Thank you for the advice. I just found it on my computer.


[WARN  Sat May 20 22:50:48 2023 82184934126041] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Mon May 22 10:15:36 2023 2.0967276151646e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[ERROR Mon May 22 10:18:12 2023 2.0982903194558e+14] ...e/nvim/site/pack/packer/start/packer.nvim/lua/packer.lua:1022: Failure running setup function: "vim/_editor.lua:0: :source (no file)..nvim_exec2() called at :source (no file):0: Vim(colorscheme):E185: Cannot find color scheme 'rose-pine'"
[ERROR Mon May 22 10:27:44 2023 2.1040134018313e+14] ...e/nvim/site/pack/packer/start/packer.nvim/lua/packer.lua:1022: Failure running setup function: "vim/_editor.lua:0: :source (no file)..nvim_exec2() called at :source (no file):0: Vim(colorscheme):E185: Cannot find color scheme 'rose-pine'"
[WARN  Mon May 22 10:42:51 2023 2.1130809928758e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:837: rose-pine/neovim wasn't updated; can't revert!
[WARN  Mon May 22 10:42:58 2023 2.1131522583921e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:837: rose-pine/neovim wasn't updated; can't revert!
[WARN  Mon May 22 10:43:00 2023 2.1131723046246e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:837: rose-pine/neovim wasn't updated; can't revert!
[WARN  Mon May 22 10:43:01 2023 2.1131753860475e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Mon May 22 10:43:01 2023 2.1131796296996e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Mon May 22 10:43:01 2023 2.1131816801938e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Mon May 22 10:43:02 2023 2.1131835374392e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Mon May 22 10:43:02 2023 2.1131849687858e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Mon May 22 10:57:32 2023 2.1218835714567e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[ERROR Mon May 22 11:14:13 2023 2.1318982143233e+14] ...e/nvim/site/pack/packer/start/packer.nvim/lua/packer.lua:1022: Failure running setup function: "vim/_editor.lua:0: :source (no file)..nvim_exec2() called at :source (no file):0: Vim(colorscheme):E185: Cannot find color scheme 'rose-pine'"
[WARN  Mon May 22 11:15:55 2023 2.1329221269783e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Wed May 24 11:23:17 2023 3.8653383868771e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Wed May 24 11:23:17 2023 3.8653384171471e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Wed May 24 11:24:52 2023 3.8662842036508e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Wed May 24 11:24:52 2023 3.8662842200608e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Wed May 24 11:27:46 2023 3.8680319487e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Wed May 24 11:27:46 2023 3.8680319704192e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[ERROR Wed May 24 12:35:25 2023 3.9086141480204e+14] ...e/nvim/site/pack/packer/start/packer.nvim/lua/packer.lua:1022: Failure running setup function: "vim/_editor.lua:0: :source (no file)..nvim_exec2() called at :source (no file):0: Vim(colorscheme):E185: Cannot find color scheme 'rose-pine'"
[ERROR Wed May 24 12:36:08 2023 3.9090512257046e+14] ...e/nvim/site/pack/packer/start/packer.nvim/lua/packer.lua:1022: Failure running setup function: "vim/_editor.lua:0: :source (no file)..nvim_exec2() called at :source (no file):0: Vim(colorscheme):E185: Cannot find color scheme 'rose-pine'"
[WARN  Wed May 24 14:34:15 2023 3.9799209884417e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Wed May 24 14:34:17 2023 3.9799397418504e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Wed May 24 14:34:18 2023 3.9799500814929e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Wed May 24 14:34:19 2023 3.9799545827633e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Wed May 24 14:34:19 2023 3.97995652025e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Wed May 24 14:34:19 2023 3.9799583258458e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Wed May 24 14:34:19 2023 3.9799601316529e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Wed May 24 14:34:20 2023 3.9799661409446e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[WARN  Wed May 24 14:37:23 2023 3.981798068465e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:837: nvim-telescope/telescope.nvim wasn't updated; can't revert!
[WARN  Wed May 24 14:37:26 2023 3.9818261218796e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:837: nvim-telescope/telescope.nvim wasn't updated; can't revert!
[WARN  Wed May 24 14:37:26 2023 3.9818336266204e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:837: nvim-telescope/telescope.nvim wasn't updated; can't revert!
[WARN  Wed May 24 14:37:29 2023 3.981859870075e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:837: nvim-telescope/telescope.nvim wasn't updated; can't revert!
[WARN  Thu May 25 11:57:27 2023 4.7498383747558e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 11:59:36 2023 4.7511270067375e+14] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:612: Potential breaking change in commit 6aeb98d of nvim-lua/plenary.nvim
[WARN  Thu May 25 12:26:30 2023 4.7672742850667e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 12:28:35 2023 4.7685184280467e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 12:52:01 2023 4.7825752847358e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 12:54:20 2023 4.7839719515338e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:22:35 2023 4.9449157244058e+14] .../site/pack/packer/start/packer.nvim/lua/packer/clean.lua:79: Cleaning cancelled!
[WARN  Thu May 25 17:23:36 2023 4.9455268216121e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:44:36 2023 4.9581305950329e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:44:36 2023 4.9581306442904e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:44:36 2023 4.9581306488117e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:44:36 2023 4.9581306566417e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:44:36 2023 4.95813066438e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:44:36 2023 4.958130674075e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:44:36 2023 4.9581306872879e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:46:39 2023 4.9593639122763e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:46:39 2023 4.9593639361121e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:46:39 2023 4.9593639626075e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:46:39 2023 4.9593639717933e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:46:39 2023 4.9593639834833e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:46:39 2023 4.9593639936142e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:48:29 2023 4.9604569338154e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:48:29 2023 4.9604569637033e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:48:29 2023 4.9604569705654e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:48:29 2023 4.9604569761775e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:48:29 2023 4.9604569807871e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:48:29 2023 4.9604569856038e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:51:30 2023 4.9622659854788e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:51:30 2023 4.9622660294788e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:51:32 2023 4.9622856923696e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:52:58 2023 4.9631540311608e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:52:58 2023 4.9631540633979e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:54:00 2023 4.963773463455e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Thu May 25 17:54:00 2023 4.9637734855263e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Fri May 26 21:31:52 2023 5.9584949321042e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Fri May 26 21:33:04 2023 5.9592087759563e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Fri May 26 21:34:30 2023 5.9600738775296e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 08:57:34 2023 6.3699113601621e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 09:10:23 2023 6.3776009894108e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:42:32 2023 6.4688952024375e+14] .../site/pack/packer/start/packer.nvim/lua/packer/clean.lua:79: Cleaning cancelled!
[WARN  Sat May 27 11:43:32 2023 6.4694953396842e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:43:32 2023 6.4694953586413e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:43:32 2023 6.4694953661458e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:55:22 2023 6.4765924788058e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:55:22 2023 6.4765925081067e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:55:22 2023 6.4765925172658e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:55:22 2023 6.4765925259454e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:58:38 2023 6.4785545523325e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 11:58:38 2023 6.4785545792133e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 12:24:44 2023 6.4942090466692e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 12:43:12 2023 6.5052921654454e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 12:44:39 2023 6.5061613133183e+14] ...m/site/pack/packer/start/packer.nvim/lua/packer/jobs.lua:94: Killing git due to timeout!
[WARN  Sat May 27 22:53:31 2023 6.871483289765e+14] ...packer/start/packer.nvim/lua/packer/plugin_types/git.lua:123: Wildcard expansion did not found any tag for plugin L3MON4D3/LuaSnip: defaulting to latest commit...
[WARN  Sun May 28 23:56:50 2023 7.7734663233813e+14] ...packer/start/packer.nvim/lua/packer/plugin_types/git.lua:123: Wildcard expansion did not found any tag for plugin L3MON4D3/LuaSnip: defaulting to latest commit...
[WARN  Fri Jun  9 23:11:38 2023 1.8582354839031e+15] ...ite/pack/packer/start/packer.nvim/lua/packer/display.lua:730: Unable to find the diff for this line
[ERROR Fri Jun  9 23:33:54 2023 1.8595716177963e+15] ...e/nvim/site/pack/packer/start/packer.nvim/lua/packer.lua:1022: Failure running setup function: "[string \":source (no file)\"]:16: attempt to call global 'ose' (a nil value)"

**Reply by L3MON4D3** [reply]
MHMMM the last one seems to be relevant.. any references to `ose` in your config?

**Reply by ZichengWangdrzz** [reply]
Thank you. It's fixed. What I did was commenting out the usage and do `:PackerSync`, then reinstall it again using `:PackerSync`. It worked!

**Reply by L3MON4D3** [reply]
Ahh nice :+1:

**Reply by no-vici** [reply]
> Ahh nice +1

Thank you for your patience and kindness as well, on behalf of a new user ;)

Cheers.
